src/py/mat3ra/made/material.py

```diff
@@ -3,6 +3,7 @@
 from mat3ra.code.constants import AtomicCoordinateUnits, Units
 from mat3ra.code.entity import HasDescriptionHasMetadataNamedDefaultableInMemoryEntity
 from mat3ra.esse.models.material import MaterialSchema
+from mat3ra.made.utils import map_array_with_id_value_to_array
```

Comment:
I thought we are using https://github.com/Exabyte-io/made/blob/main/src/py/mat3ra/made/utils.py

Reply:
mat3ra.made.utils?